tractor/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
 project(tractor)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -std=c++17 -O3 -mavx -U__AVX__")
```

Comment:
Did you forget to add the standard directive you added above?

Reply:
done